lib/gui/components/svg-icon/svg-icon.js

```diff
@@ -21,8 +21,94 @@
  */
 
 const angular = require('angular');
+const react = require('react');
+const propTypes = require('prop-types');
+const react2angular = require('react2angular').react2angular;
+const path = require('path');
+const fs = require('fs');
+const _ = require('lodash');
+
 const MODULE_NAME = 'Etcher.Components.SVGIcon';
-const SVGIcon = angular.module(MODULE_NAME, []);
-SVGIcon.directive('svgIcon', require('./directives/svg-icon'));
+const angularSVGIcon = angular.module(MODULE_NAME, []);
+
+/**
+ * @summary SVG element that takes both filepaths and file contents
+ * @type {Object}
+ * @public
+ */
+class SVGIcon extends react.Component {
+
+  /**
+   * @summary Render the SVG
+   * @returns {react.Element}
+   */
+  render() {
+    let contents = null;
+
+    // The path contains SVG contents
+    if (_.first(this.props.path) === '<') {
```

Comment:
...but if an SVG with leading whitespace is technically malformed (I dunno, I've never looked at the standard), then we should probably reject it, rather than trying to "massage" it back into a valid SVG?